### PR TITLE
Fix project card headers

### PR DESCRIPTION
**Pull request by maciaszczykm**
### What this PR does / why we need it
Previously longer names with `-` in it were automatically wrapped and this was causing some visual glitches. I've set `nowrap` and switched to CSS method as it is more stable (not all letters have the same width and this sets limit on div size not letter count).

Zrzut ekranu 2021-04-28 o 08 55 39

### Which issue(s) this PR fixes
Fixes https://github.com/kubermatic/dashboard/issues/3271.

### Release note

```release-note
NONE
```

### CODE REVIEW

**Review by kgroschoff — COMMENTED** [review]
Great!
/lgtm

File: src/app/project/template.html [hunk 1]
```diff
@@ -209,7 +209,10 @@
            class="km-health-state km-project-cards-status"
            fxFlexAlign=" center"></i>
         <mat-card-title [matTooltip]="getProjectTooltip(project.name)"
-                        [attr.id]="'km-project-name-' + project.name">{{getName(project.name)}}</mat-card-title>
+                        [attr.id]="'km-project-name-' + project.name"
+                        class="card-header">
```
Comment (floreks) [inline src/app/project/template.html:213]: I do not see this `card-header` class definition. Where is it?

Comment (maciaszczykm) [reply src/app/project/template.html:213]: It's a leftover. PTAL

**Pull request merged**